proto/proto-backplane-grpc/src/main/proto/deephaven/proto/table.proto

```diff
@@ -315,8 +315,14 @@ message EmptyTableRequest {
 
 message TimeTableRequest {
   Ticket result_id = 1;
-  sint64 start_time_nanos = 2 [jstype=JS_STRING];
-  sint64 period_nanos = 3 [jstype=JS_STRING];
+  oneof start_time {
+    uint64 start_time_nanos = 2 [jstype = JS_STRING];
```

Comment:
```suggestion
    sint64 start_time_nanos = 2 [jstype = JS_STRING];
```

this may be a backwards incompatible change?

Reply:
Agreed. I thought an earlier version of this RPC was "correct", but maybe I didn't review close enough.

Reply:
Confirmed, go tests will pass with the old generated code if you do this, once #4211 merges.